app/grandchallenge/core/shortcuts.py

```diff
@@ -0,0 +1,16 @@
+from functools import partial
```

Comment:
Would be clearer if this was `guardian.py` and also included `PermissionListMixin`, and `ObjectPermissionRequiredMixin` with `accept_global_perms = False` set explicitly, just in case they change it upstream.